metadata.txt

```diff
@@ -23,8 +24,9 @@ email=olivier.dalang@gmail.com
 
 # Uncomment the following line and add your changelog entries:
 changelog=- 2014-05-22 - Version 0.0 : intial release
-- 2014-05-26 - Version 0.1 : linear and uniform transformation method, scipy dependency no more needed, better management of pinned points
```

Comment:
Is this reformatting necessary for Qgis 3 compatibility ? If not, please revert.

If you wish to improve the style, that's welcome, but should be part of another PR (so that this one really stays about porting to QGIS3)